### PR TITLE
Remove docopt

### PR DESCRIPTION
**Pull request by Tal-or**
Identical to changes that were done on u/s version k8stopologyawareschedwg/resource-topology-exporter#23
plus the fixes from k8stopologyawareschedwg/resource-topology-exporter#25

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -83,3 +83,7 @@ undeploy:
 gen-manifests:
 	@curl -L $(TOPOLOGYAPI_MANIFESTS)/crd.yaml
 	@hack/get-manifest-ds.sh
+
+.PHONY: update-golden-files
+update-golden-files:
+	@go test ./cmd/... -update
```
Comment (fromanirh) [inline Makefile:89]: let's discuss in u/s if we need this or not (or some changes here in general)

**Review by fromanirh — COMMENTED** [review]
the failure is expected unfortunately, still working to fix it

**Comment by fromanirh** [comment]
looks good, thanks. But we don't have tests running there, so let's figure out how we can prevent regressions.
Adding some e2e sanity tests is a option, but we need to do that in a separate PR (obviously)

**Comment by Tal-or** [comment]
> looks good, thanks. But we don't have tests running there, so let's figure out how we can prevent regressions. Adding some e2e sanity tests is a option, but we need to do that in a separate PR (obviously)

We have unittest running under ```main_test.go``` is that not good enough? most of the e2e test are covered in the repo under the working group

**Review by fromanirh — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**